src/ray/util/event.cc

```diff
@@ -239,6 +239,23 @@ bool RayEvent::IsLevelEnabled(rpc::Event_Severity event_level) {
 
 void RayEvent::SetLevel(const std::string &event_level) { SetEventLevel(event_level); }
 
+RayLogLevel RayEvent::EventLevelToLogLevel(const rpc::Event_Severity &severity) {
+  switch (severity) {
+  case rpc::Event_Severity_INFO:
+    return RayLogLevel::INFO;
+  case rpc::Event_Severity_WARNING:
+    return RayLogLevel::WARNING;
+  case rpc::Event_Severity_ERROR:
+  // Converts fatal events to error logs because fatal logs will lead to process exiting
+  // directly.
+  case rpc::Event_Severity_FATAL:
+    return RayLogLevel::ERROR;
+  default:
+    RAY_LOG(FATAL) << "Can't cast severity " << severity;
```

Comment:
this is a bit aggressive. Should we use error instead?

Reply:
OK